getting-started/what-you-need.md

```diff
@@ -11,58 +11,50 @@ position: 3
 
 # What You Need to Use the Telerik Blazor Components
 
-This article explains the packages you need to use the Telerik UI for Blazor components, how to get them, and how to configure your project to include the Telerik Blazor components.
+This article describes the packages that provide the Telerik UI for Blazor components, how to get them, and how to manually configure your project to use them.
 
-This article is a shorter, more focused version of the Getting Started articles and targets people who are familiar with the Telerik NuGet Feed and Blazor in general. If you are looking for step-by-step tutorials, see the [Client-side Blazor - Tutorial]({%slug getting-started/client-side%}) and [Server-side Blazor - Tutorial]({%slug getting-started/server-side%}) articles.
```

Comment:
The second and the third paragraph provide hints that aren't directly related to this article. Adding them as tips helps the reader to visually identify such additional information and skip it if needed.

Reply:
The first paragraph is not completely accurate technically now - there is a bunch of packages linked here that don't actually carry components - the Recurrence and DataSource packages are helper code, and the DPL is also not components. To some extent, all of them are mandatory, but we have taken care of the required references internally.

That said, I am not sure I want the old wording back.